### PR TITLE
make units tests independant

### PR DESCRIPTION
**Pull request by maathor**

### CODE REVIEW

File: tests/aws/aws_kubernetes.rs [hunk 1]
```diff
@@ -259,97 +149,97 @@ fn create_eks_cluster_in_eu_west_3() {
...
+     }
+
+     let _ = match tx.commit() {
+         TransactionResult::Ok => assert!(true),
+         TransactionResult::Rollback(_) => assert!(false),
+         TransactionResult::UnrecoverableError(_, _) => assert!(false),
+     };
+ }
+
+fn delete_eks_cluster_in_eu_west_3() {
```
Comment (deimosfr) [inline tests/aws/aws_kubernetes.rs:199]: what if the cluster is not deployed? wouldn't it be less error prone to integrate it directly in the create_and_destroy_eks_cluster_in_eu_west_3 for example?

Comment (maathor) [reply tests/aws/aws_kubernetes.rs:199]: It s not a test

File: tests/aws/aws_kubernetes.rs [hunk 2]
```diff
@@ -165,17 +58,17 @@ fn create_eks_cluster_in_us_east_2() {
 
     let cloudflare = dns_provider_cloudflare(&context);
 
-    let mut file = File::open("tests/assets/eks-options.json").unwrap();
+    let mut file = File::open("cloned-engine/tests/assets/eks-options.json").unwrap();
```
Comment (deimosfr) [inline tests/aws/aws_kubernetes.rs:61]: wait why? there is no cloned-engine in this repo

File: tests/aws/aws_kubernetes.rs [hunk 3]
```diff
@@ -259,97 +149,97 @@ fn create_eks_cluster_in_eu_west_3() {
...
-//         _ => {}
-//     }
-//
-//     let _ = match tx.commit() {
-//         TransactionResult::Ok => assert!(true),
-//         TransactionResult::Rollback(_) => assert!(false),
-//         TransactionResult::UnrecoverableError(_, _) => assert!(false),
-//     };
-// }
+fn delete_eks_cluster_in_us_east_2() {
```
Comment (deimosfr) [inline tests/aws/aws_kubernetes.rs:152]: Could you just add a comment message to say it's only for testing purpose (tooling)

**Review by deimosfr — COMMENTED** [review]
+ remove 1 or 2 warning plz ;)

File: tests/aws/aws_kubernetes.rs (hunk 3 above)
Comment (maathor) [reply tests/aws/aws_kubernetes.rs:152]: already done

File: tests/aws/aws_kubernetes.rs (hunk 2 above)
Comment (maathor) [reply tests/aws/aws_kubernetes.rs:61]: oups

**Review by deimosfr — APPROVED** [review]

**Pull request merged**